### ISSUE TITLE
🐞 `dagger.exe` has no output. v0.11.0. Windows.

### ISSUE THREAD

**Issue by jpadams** [dagger/dagger#7045]
### What is the issue?

I installed `dagger.exe` and ran `dagger.exe` and got no output.

<img width="847" alt="image" src="https://github.com/dagger/dagger/assets/3187222/e6805235-2992-4412-bd60-d58ca774c86a">

### Dagger version

no ouput, but dagger v0.11.0 amd64 Windows

### Steps to reproduce

Go to user's home directory.

run
`Invoke-WebRequest -UseBasicParsing -Uri https://dl.dagger.io/dagger/install.ps1 | Invoke-Expression`

execute
`./dagger/dagger.exe`

### Log output

<nothing>

**Comment by fcbry** [comment]
Checked out and attempted to build from main 

```
commit d3a11cf89ec9685d4c60737d3ab2af7bd4e558e0 (HEAD -> main, origin/main, origin/HEAD)
Author: Helder Correia <174525+helderco@users.noreply.github.com>
Date:   Thu Apr 11 11:17:59 2024 +0000
```
```
❯ cd .\cmd\dagger ; go install ; go build . ; dagger.exe --help

dagger\cmd\dagger on  main via 🐹 v1.22.2 took 3s
```

(no output)

**Comment by levlaz** [comment]
@jpadams and I did a bit of pairing and were able to figure some things out. 

It looks like the issue is with the new TUI + windows, specifically when `--progress-auto` by default you get no output in windows. However, if you switch to `--progress=plain` then we are able to see all the right output. 

`init` still does not work regardless of the output settings due to the issues covered here: https://github.com/dagger/dagger/pull/7003 

```
$ ./dagger.exe --progress plain version
dagger v0.11.0 (registry.dagger.io/engine) windows/amd64
```

**Comment by pjmagee** [comment]
I thought it was me. I tried everything, nothing seems to work. I checked my paths, tried to run it from Powershell, Cmd etc. This is a big issue for the tutorial

**Comment by pjmagee** [comment]
A "workaround" for now could be to use WSL?

**Comment by fcbry** [comment]
> A "workaround" for now could be to use WSL? I managed to get it working fine on Ubuntu WSL. WSL saves the day 🗡️

Yes WSL is an option, but hopefully it gets resolved, only working on *nix, makes it a bit of a harder sell. I'd be willing to help in my free time with setting github windows builds (1/4 century nix, but working in a windows shop now).

**Comment by fcbry** [comment]
thank you @jedevc
